shrine_with_cart/mdc_100_series/lib/short_bottom_sheet.dart

```diff
@@ -0,0 +1,631 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart' show timeDilation;
+import 'package:flutter/services.dart';
+import 'package:meta/meta.dart';
+import 'package:scoped_model/scoped_model.dart';
+
+import 'colors.dart';
+import 'model/app_state_model.dart';
+import 'model/product.dart';
+import 'shopping_cart.dart';
+
+class ShortBottomSheet extends StatefulWidget {
+  final AnimationController hideController;
+  const ShortBottomSheet({Key key, this.hideController}) : super(key: key);
```

Comment:
I think you need to put the constructor before the property.

Reply:
Makes sense, done